client/jetpack-cloud/sections/agency-dashboard/dashboard-bulk-actions/index.tsx

```diff
@@ -29,22 +38,36 @@ export default function DashboardBulkActions( { selectedSites }: Props ) {
 		},
 	];
 
+	const otherMonitorActions = [
+		{
+			label: translate( 'Custom Notification' ),
+			action: () => toggleNotificationSettingsPopup(),
+		},
+	];
+
```

Comment:
Is there any reason why this isn't part of `toggleMonitorActions` (probably with another name)?

Reply:
The reason behind this is to keep the toggle actions(Pause & Resume Monitor) together and other actions like Custom Notification and Reset Notification in a different variable so that we can group the toggle actions together. 

This is what we are doing. 

```
<ButtonGroup>
	{ toggleMonitorActions.map( ( { label, action } ) => (
		<Button key={ label } onClick={ action }>
			{ label }
		</Button>
	) ) }
</ButtonGroup>
{ otherMonitorActions.map( ( { label, action } ) => (
	<ButtonGroup key={ label }>
		<Button onClick={ action }>{ label }</Button>
	</ButtonGroup>
) ) }
```